### ISSUE TITLE
Not working in android emulator

### ISSUE THREAD

**Issue by vijayst** [robinpowered/react-native-fetch-polyfill#6]
I created a [minimum example for react-native-fetch-polyfill](https://github.com/vijayst/fetch-demo). I set a timeout of 5 seconds. The timeout worked in iOS simulator but not working in Android emulator. In Android emulator, fetch times out only after 60 seconds.

**Comment by vijayst** [comment]
Looks like it is a problem with react-native android. Opened an issue in react-native repo: https://github.com/facebook/react-native/issues/14134

**Comment by techrah** [comment]
What about on Android devices (not simulator)? Also broken?

**Comment by vijayst** [comment]
@ryanhomer It is working fine on Android devices.

**Comment by pstanton** [comment]
not working on android device over adb. tried with chrome debugger on and off.

**Comment by atticoos** [comment]
@pstanton what version of React Native are you running, and what device are you using that reproduced this problem?

**Comment by pstanton** [comment]
https://github.com/robinpowered/react-native-fetch-polyfill/issues/7#issuecomment-364006398

**Comment by terreb** [comment]
Wow, the issue was created quite some time ago, but still not fixed! Any update?

**Comment by joeythelantern** [comment]
Still doesn't work on Android Simulator.  +1

**Comment by zachdunn** [comment]
> Wow, the issue was created quite some time ago, but still not fixed! Any update?

@terreb + @joeythelantern Appreciate the attentiveness. This is one of our "Help Wanted" tagged issues, and we'd be happy to review any pull requests you'd like to submit for this one.

**Comment by ktownsend-cbre** [comment]
I just ran into this problem. I'm hoping this isn't broken on device, but can't confirm that behavior until tomorrow.

**Comment by TuurDutoit** [comment]
As per [my comment](https://github.com/robinpowered/react-native-fetch-polyfill/issues/7#issuecomment-518732215) on another issue in this repo, this makes sense. The discrepancy between the Android and iOS implementations is probably the culprit.
